apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/db/derived/EntityDbTable.java

```diff
@@ -388,6 +396,42 @@ public void insert(T t) {
                     pstmt.executeUpdate();
                 }
             }
+            if (supportDelete()) {
```

Comment:
More explanations in comments are needed for the code below.

Reply:
Added